lib/macho/load_commands.rb

```diff
@@ -171,6 +27,35 @@ def self.new_from_bin(view)
       self.new(view, *bin.unpack(format))
     end
 
+    # Creates a new (viewless) command corresponding to the symbol provided
+    # @param cmd_sym [Symbol] the symbol of the load command being created
+    # @param args [Array] the arguments for the load command being created
+    # @note this does not currently work. DO NOT USE IT.
+    def self.create(cmd_sym, cmd_round, *args)
```

Comment:
This is probably obvious from my previous comments, but I don't think we should hard-code the required alignment on creation. All target-specific encoding should really only happen in the `serialize` method. This will allow creation of a target-agnostic load command that can be then serialized to all Mach-O slices of a given fat binary, e.g. when adding an `LC_RPATH` command to a fat binary. But really the point here is that I'd like the load command to not make any assumptions about how it is going to be serialized until it actually gets serialized.